File: Pandemic/Disease.java

/**
 * Write a description of class Disease here.
 * 
 * @author (your name) 
 * @version (a version number or a date)
 */
public class Disease
{
    // instance variables - replace the example below with your own
    public String colour;
    public boolean cured;
    public boolean eliminated;


    /**
     * Constructor for objects of class Disease
     */
    public Disease(String diseaseColour)
    {
        colour = diseaseColour;
        cured = false;
        eliminated = false;
        
    }
    
    public boolean getCured()
    {
        return cured;
    }
    
    public void setCured(boolean toSet)
    {
        cured = toSet;
    }

    public void setColour(String newColour)
    {
        colour = newColour;
    }
    
    public String getColour()
    {
        return colour;
    }
    
    
}

File: disease.py
# coding: utf-8


class Disease:
    def __init__(self, colour):
        self.colour = colour
        self.cured = False

    def __repr__(self):
        return '{}({!r})'.format(
            self.__class__.__name__, self.colour)

    def __str__(self):
        result = '{} disease'.format(self.colour)
        if self.cured:
            result += ' (cured)'

        return result

